doc/static_discovery_configuration_guide.md

```diff
@@ -0,0 +1,808 @@
+```
+
+**Note:** Use the password (`tls_truststore_password`) created in step 3.6.
+
+Set the file permissions of the new `/etc/harmony-ap/clientauthentication.xml` file:
+
+```
+sudo chown harmony-ap:harmony-ap /etc/harmony-ap/clientauthentication.xml
+sudo chmod 751 /etc/harmony-ap/clientauthentication.xml
+```
+
+### 3.8 Apply Changes
+
+This step requires shell access to the host.
+
+On Access Points 1 (`org1_gw`) and Access Point 2 (`org2_gw`), restart the `harmony-ap` service to apply the 
+configuration changes:
+
+```
+sudo systemctl restart harmony-ap
+```
+
+### 3.9 Send Test Message
+
+Send a request to Access Point 1 (`org1_gw`) using the curl command below. The request (`submitRequest.xml`) can be downloaded [here](configuration_examples/static_discovery/submitRequest.xml). The content inside the payload's `value` element must be [base64 encoded](https://www.base64encode.org/).
+
+```
+curl -u org1:<org1_plugin_user_password> --header "Content-Type: text/xml;charset=UTF-8" --data @submitRequest.xml https://<AP1_IP_OR_FQDN>:8443/services/backend -v -k
+```
+
+A successful response looks like this:
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<soap:Envelope xmlns:soap="http://www.w3.org/2003/05/soap-envelope">
+   <soap:Body>
+      <ns2:submitResponse xmlns:ns2="http://org.ecodex.backend/1_1/" xmlns:ns3="http://docs.oasis-open.org/ebxml-msg/ebms/v3.0/ns/core/200704/" xmlns:ns4="http://www.w3.org/2003/05/soap-envelope" xmlns:xmime="http://www.w3.org/2005/05/xmlmime">
+         <messageID>bd3ea132-6a0d-11ec-9789-0af9f1d3371a@edelivery.digital</messageID>
+      </ns2:submitResponse>
+   </soap:Body>
+</soap:Envelope>
+```
+
+List received and pending messages on the Access Point 2 (`org2_gw`). The request (`listPendingMessagesRequest.xml`) can be downloaded [here](configuration_examples/static_discovery/listPendingMessagesRequest.xml).
+
+```
+curl -u org2:<org2_plugin_user_password> --header "Content-Type: text/xml;charset=UTF-8" --data @listPendingMessagesRequest.xml https://<AP2_IP_OR_FQDN>:8443/services/backend -v -k
+```
+
+A successful response looks like this:
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<soap:Envelope xmlns:soap="http://www.w3.org/2003/05/soap-envelope">
+   <soap:Body>
+      <ns2:listPendingMessagesResponse xmlns:ns2="http://org.ecodex.backend/1_1/" xmlns:ns3="http://docs.oasis-open.org/ebxml-msg/ebms/v3.0/ns/core/200704/" xmlns:ns5="http://www.w3.org/2003/05/soap-envelope" xmlns:xmime="http://www.w3.org/2005/05/xmlmime">
+         <messageID>bd3ea132-6a0d-11ec-9789-0af9f1d3371a@edelivery.digital</messageID>
+      </ns2:listPendingMessagesResponse>
+   </soap:Body>
+</soap:Envelope>
+```
+
+Retrieve the test message from Access Point 2 (`org2_gw`). The request (`retrieveMessageRequest.xml`) can be downloaded [here](configuration_examples/static_discovery/retrieveMessageRequest.xml). Before sending the message, replace the `MESSAGE_ID` placeholder with the ID (`messageID`) of the test message.
```

Comment:
Raw link comment also applies here.

Reply:
I updated the link to point to the raw file.